spec/spec_helper.rb

```diff
@@ -128,4 +99,37 @@ def with_minitest_loaded
   end
 end
 
+RSpec.configure do |config|
+  config.color = true
+  config.order = :random
+
+  config.include CommonHelperMethods
+  config.include RSpec::Support::InSubProcess
+
+  config.expect_with :rspec do |expectations|
+    $default_expectation_syntax = expectations.syntax # rubocop:disable Style/GlobalVars
+    expectations.syntax = :expect
+    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+    expectations.strict_predicate_matchers = true
+  end
+
+  config.mock_with :rspec do |mocks|
+    mocks.verify_partial_doubles = true
+  end
+
+  config.include_context "with #should enabled", :uses_should
```

Comment:
Because they are included here now.

Reply:
Only these need to be below, the rest can stay up top.